### DISCUSSION TITLE
privacytests.org results

### DISCUSSION THREAD

**Discussion by arthuredelstein** [Eloston/ungoogled-chromium#1793] [Show and tell]
Hi everyone -- Happy New Year! I am pleased to share the results of privacy tests for Ungoogled Chromium on desktop:

https://privacytests.org

I'll be happy to answer any questions about these tests!

**Comment by kramred** [comment]
**First:**
It's okay it you're a fan of the "Brave" browser. 

**Second:**
Making such a project/page where it's presented as if "Brave" was more privacy friendly than e.g. the Tor browser is a bit dodgy so say the least.

---

To anyone who doesn't spend a considerable amount of time trying to understand what data browsers send here's my 2 cents:

**Generally:** 
These kinds of comparisons (where one contestant seems to have a significant advantage over all others) should be interpreted with caution. Most of the time they turn out to be a (hidden) advertisement, i.e. they have a different agenda than they claim.

**Here:** 
See [Objectives](https://github.com/Eloston/ungoogled-chromium#objectives) of ungoogled-chromium: 
>  1. sans dependency on Google web services
>  2. retains the default Chromium experience as closely as possible
>  3. tweaks to enhance privacy, control, and transparency
> where the objectives conflict, the objective of higher significance should take precedence

That means that ungoogled-chromium without extensions such a [uBlock Origin](https://github.com/gorhill/uBlock) or [uMatrix](https://github.com/gorhill/uMatrix) is rather close to the default Chromium experience. So you have to take extra steps to reduce the amount of data that is send.

[Here's](https://www.kuketz-blog.de/brave-datensendeverhalten-desktop-version-browser-check-teil1/) a review of the "Brave" browser in German where they state that "Brave" leaves a mixed impression.
> Overall, Brave leaves a mixed impression. The browser's default settings are not ideal. Among other things, Google as the default search engine, user tracking and the reloading of JavaScript from various Google, Facebook and Twitter domains are to be criticized in order not to impair the functionality of the platforms (social media). The track protection should actively ask whether the user wants to allow or deny the integration.
> ...
> It remains unclear whether one belongs to the A/B test group and which data is sent to the domain `variations.brave.com`. The query of affiliate domains via the domain `laptop-updates.brave.com` is also questionable.
> …
> The external presentation and the promise of privacy protection do not quite match the actual behavior in practice.

**Provisional conclusion:** 
Maybe all those green check-marks apply to "Brave"; does that make "Brave" the most privacy friendly browser? Possibly not.

One would have to check 

• what all these tests actually mean for tracking you or leaking data that you did not want to send 
• what other data is sent (e.g. to services controlled by "Brave")

In general it's about what your expectations are regarding the (extra) data your browser sends and how well these are met by the browser you're using.

**Reply by arthuredelstein** [reply]
> **Generally:** These kinds of comparisons (where one contestant seems to have a significant advantage over all others) should be interpreted with caution. Most of the time they turn out to be a (hidden) advertisement, i.e. they have a different agenda than they claim.

Not this time. Privacytests.org is an independent project. I have not received funding from anyone for this project; if I do in the future I will report it. I have endeavored to make the tests objective and I have no intention of favoring any browser. There is no hidden agenda or advertisement.

That said, I welcome questions or criticism. The project is open source, so you can examine how it reached the conclusions it did.

> **Here:** See [Objectives](https://github.com/Eloston/ungoogled-chromium#objectives) of ungoogled-chromium:
> 
> > 1. sans dependency on Google web services
> > 2. retains the default Chromium experience as closely as possible
> > 3. tweaks to enhance privacy, control, and transparency
> >    where the objectives conflict, the objective of higher significance should take precedence

I expect, with some work, ungoogled-chromium could get mostly green checkmarks (except perhaps in the tracking blocking section) while remaining consistent with the objectives. I do plan to expand the categories to examine what kind of data is "sent home" by web browsers, which would likely be reflected in your efforts in (1).

**Comment by ghost** [comment]
Any updates so far on this? Ungoogled Chromium is failling on some important tests and Brave stills the most succeed one.
